packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -61,7 +61,7 @@ impl PersistedOntologyType for PersistedLinkType {
             PersistedOntologyIdentifier::new(link_type.record.id().clone(), link_type.account_id);
         Self {
             inner: link_type.record,
-            identifier,
+            metadata: PersistedOntologyMetadata { identifier },
```

Comment:
You're able to do this all over the place because you set identifier to `pub`. Can we change this and use the `::new` constructor?

Reply:
Fixed in https://github.com/hashintel/hash/pull/1199/commits/396765840d31f7cc5dc9af2feb3315e0aecfdc82